redaxo/src/core/lang/de_de.lang

```diff
@@ -196,12 +196,12 @@ setup_311 = Die folgenden Dateien/Verzeichnisse sind für alle Anwender auf dem
 setup_312 = Schritt 3 erneut durchführen
 setup_399 = 3 / Systemcheck
 
-setup_400 = Setup: Schritt 4 von 7 / Schreiben der 'data/config.yml'
-setup_401 = data/config.yml konnte nicht geschrieben werden.
+setup_400 = Setup: Schritt 4 von 7 / Schreiben der '{0}'
+setup_401 = {0} konnte nicht geschrieben werden.
 setup_402 = Allgemeines
 setup_403 = Datenbankinformationen
 setup_404 = Die angegebene Datenbank existiert bereits!
-setup_405 = Schreiben der 'data/config.yml'
+setup_405 = Schreiben der '{0}'
```

Comment:
Diese meldung scheint nirgends verwendet zu werden

Reply:
Kann dann gerne weg. (am besten aber auch in allen Sprachen)